deploy/runtools/runtime_config.py

```diff
@@ -177,7 +197,7 @@ def array_to_lognames(values: Sequence[Optional[Any]], prefix: str) -> List[str]
         permissive_driver_args += command_linklatencies
         permissive_driver_args += command_netbws
         permissive_driver_args += command_shmemportnames
-        driver_call = f"""sudo ./{driver} +permissive {" ".join(permissive_driver_args)} +permissive-off {" ".join(command_bootbinaries)}"""
+        driver_call = f"""sudo LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH ./{driver} +permissive {" ".join(permissive_driver_args)} {extra_plusargs} +permissive-off {" ".join(command_bootbinaries)} {extra_args} """
```

Comment:
Maybe we should just put "." on the rpath instead?

Reply:
We can push this one to later if think it's a good change.

Reply:
good call, fixed rpath setup on vcs/verilator builds and removed LD_LIBRARY_PATH addition